lib/puppet/provider/base_dsc/powershell.rb

```diff
@@ -2,6 +2,8 @@
   confine :operatingsystem => :windows
   defaultfor :operatingsystem => :windows
 
+  require 'lib/puppet/file_system'
```

Comment:
Yep. Just setting it first where I know there isn't an issue and will be moving it once things are rolling